### PR TITLE
fix #55891 sankebiz.jp

### LINKED ISSUE #55891: sankeibiz.jp
### Issue URL (Annoyance)
 [http://www.sankeibiz.jp/smp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.sankeibiz.jp%2Fsmp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | 3.5 nightly 1 (5)
Browser: | Chrome (data compression)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55891

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
 gyao.yahoo.co.jp##.ad-yads
+sankeibiz.jp##.module_recommend:has(> div.module_content > div.OUTBRAIN )
```
Comment (Alex-302) [inline JapaneseFilter/sections/specific.txt:7]: wrong file and incorrect syntax

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:7]: This rule wasn't removed. Also it has unnecessary space after `div.OUTBRAIN` - it's incorrect syntax.

File: JapaneseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -44,6 +44,7 @@ mindhack2ch.com#?#.article-body > .rss:has(> script)
 mindhack2ch.com#?#.article-post-bottom > .rss-article:has(> .rss-article-item)
 !
 punisoku.blogo.jp#?#.plugin-memo:has(a[href*="www.dlsite"])
+sankeibiz.jp#?##wrapper > main > section.module_recommend:has(> div.module_content > div.OUTBRAIN )
```
Comment (zloyden) [inline JapaneseFilter/sections/css_extended.txt:47]: Incorrect syntax at the end.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply JapaneseFilter/sections/specific.txt:7]: still not fixed

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**